### PR TITLE
Missing events added to README, including NKBIP-01 and NKBIP-02

### PR DESCRIPTION
**Pull request by SilberWitch**
I updated the [Nostr Event Register](https://wikistr.com/nostr-event-register*dd664d5e4016433a8cd69f005ae1480804351789b59de5af06276de65633d319) to include the new entries in this repo and I am now proposing the additions from my own list, so that the two are again comprehensive.

I also corrected a wiki link that was broken.

### CODE REVIEW

**Review by fiatjaf — APPROVED** [review]

**Comment by staab** [comment]
Hmmm 1986 appears to be overloaded. I've been using it for almost a year for relay reviews. Who gets to keep their kind?

**Comment by AsaiToshiya** [comment]
Blossom seems to use `kind:10063` in [BUD-03](https://github.com/hzrd149/blossom/blob/master/buds/03.md). What is the reason for no adding it?

**Comment by SilberWitch** [comment]
We can move to 1987. No problem. Only writing test data, so far.
I'll add your 1986. Do you have a link to the spec for it, @staab?

He'd only listed the one. I'll read through his newer specs and add the missing ones.

**Comment by SilberWitch** [comment]
Nevermind. Found the 1986 PR and made a wiki page to link to.

https://wikistr.com/nip-85

**Comment by SilberWitch** [comment]
Sorry for the messy commits. Should be correct, now.

File: README.md [hunk 1]
```diff
@@ -131,6 +131,8 @@ They exist to document what may be implemented by [Nostr](https://github.com/nos
 | `1971`        | Problem Tracker                 | [nostrocket][nostrocket]               |
 | `1984`        | Reporting                       | [56](56.md)                            |
 | `1985`        | Label                           | [32](32.md)                            |
+| `1986`        | Relay reviews                   | [NIP-85]                               |
```
Comment (AsaiToshiya) [inline README.md:134]: I don't know if it is appropriate to add NIP kind that is not yet merged.

Comment (SilberWitch) [reply README.md:134]: I don't mind, if you change the link. You could link to the PR, but it is also called NIP-85.

Comment (fiatjaf) [inline README.md:134]:
```suggestion
| `1986`        | Relay reviews                   |                                        |
```

File: README.md [hunk 2]
```diff
@@ -202,10 +209,15 @@ They exist to document what may be implemented by [Nostr](https://github.com/nos
...
 
-[NUD: Custom Feeds]: https://wikifreedia.xyz/cip-01/97c70a44366a6535c1
+[NUD: Custom Feeds]: https://wikifreedia.xyz/cip-01/
 [nostrocket]: https://github.com/nostrocket/NIPS/blob/main/Problems.md
 [lnpub]: https://github.com/shocknet/Lightning.Pub/blob/master/proto/autogenerated/client.md
 [joinstr]: https://gitlab.com/1440000bytes/joinstr/-/blob/main/NIP.md
+[NKBIP-01]: https://wikistr.com/nkbip-01
+[NKBIP-02]: https://wikistr.com/nkbip-02
+[Blossom]: https://wikistr.com/blossom
+[NIP-85]: https://wikistr.com/nip-85
```
Comment (fiatjaf) [inline README.md:219]:
```suggestion
```

File: README.md (hunk 1 above)
Comment (staab) [reply README.md]: I agree, but this makes PRs second class citizens compared to wikifreedia entriea. Maybe we should have a "unmerges and third party" kinds table?

**Review by AsaiToshiya — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (AsaiToshiya) [reply README.md]: For now, fiatjaf's changes look good.

**Review by staab — APPROVED** [review]

**Pull request merged**